src/inject/dynamic-theme/css-rules.ts

```diff
@@ -17,6 +17,8 @@ export function iterateCSSRules(rules: CSSRuleList, iterate: (rule: CSSStyleRule
             } catch (err) {
                 logWarn(err);
             }
+        } else if (rule instanceof CSSSupportsRule) {
+            iterateCSSRules(rule.cssRules, iterate);
```

Comment:
Great!

Now, there is `rule.conditionText`. Would be great to iterate style when the condition is supported (e.g. when there is a condition for IE, it will be not good to iterate styles).

I've checked, there is `[CSS.supports](https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports)` function. So try something like:
```javascript
if (CSS.supports(rule.conditionText)) {
    iterateCSSRules(rule.cssRules, iterate);
}
```